pycqed/instrument_drivers/physical_instruments/_QCC/QCCMicrocode.py

```diff
@@ -0,0 +1,284 @@
+import logging
+import sys
+
+
+def is_number(s):
+    try:
+        float(s)
+        return True
+    except ValueError:
+        return False
+
+
+def get_bin(x, n, Unsigned=True):
+    '''
+    Return the 2's complement of the integer number $x$
+    for a given bitwidth $n$.
+    '''
+    if (not is_number(x)):
```

Comment:
The function description says that the number should be an integer. But the function `is_number(...)` checks if the value is a float.  This could lead to weird behaviour